uaclient/cli.py

```diff
@@ -1969,10 +1967,13 @@ def wrapper(*args, **kwargs):
 
 @main_error_handler
 def main(sys_argv=None):
+    log_file = (
+        defaults.CONFIG_DEFAULTS["log_file"]
+        if util.we_are_currently_root()
+        else pro_log.get_user_log_file()
+    )
```

Comment:
I think this logic can happen inside of `setup_logging` instead of doing it twice at this level